stac_ipyleaflet/stac_discovery/stac_widget.py

```diff
@@ -154,154 +280,271 @@ def list_palettes(add_extra=False, lowercase=False):
         )
-        buttons.style.button_width ="50px"
+        buttons_box = Box([stac_buttons], layout=layouts["buttons"])
+        stac_tab_labels = ['Catalog', 'Visualization']
+        tab_widget_children = []
+        stac_tab_widget = Tab()
+
+        for label in stac_tab_labels:     
+            tab_content = VBox()
+            if label == 'Catalog':
+                tab_content.children = [
+                    catalogs_box,
+                    collections_box,
+                    collection_description_box,
+                    collection_url_box,
+                    collection_dates_box,
+                    items_box
+                ]
+            elif label == 'Visualization':
+                tab_content.children = [
+                    raster_options
+                ]
+            tab_widget_children.append(tab_content)
+        stac_tab_widget.children = tab_widget_children
+        stac_tab_widget.titles = stac_tab_labels
         stac_widget.children = [
-            catalogs,
-            collection,
-            collection_description,
-            collection_url,
-            HBox([start_date, end_date]),
-            items,
-            raster_options,
-            buttons,
+            # catalogs_box,
+            # collections_box,
+            # collection_description_box,
+            # collection_url_box,
+            # collection_dates_box,
+            # items_box,
+            # palettes_dropdown,
+            # raster_options,
+            stac_tab_widget,
+            buttons_box,
             output
         ]
+    
 
-        # Event Watchers
-        def catalogs_changed(change):
-            if change["new"]:
-                df = pd.read_csv(stac_info[catalogs.value]["filename"], sep="\t")
-                datasets = df[stac_info[catalogs.value]["name"]].tolist()
-                collection.options = datasets
-                collection.value = datasets[0]
-
-        catalogs.observe(catalogs_changed, names="value")
-
-        def collection_changed(change):
-            if change["new"]:
-                output.clear_output()
-                df = pd.read_csv(stac_info[catalogs.value]["filename"], sep="\t")
-                df = df[df[stac_info[catalogs.value]["name"]] == collection.value]
-                collection_description.value = df[stac_info[catalogs.value]["description"]].tolist()[0]
-                collection_url.value = df[stac_info[catalogs.value]["url"]].tolist()[0]
+        def update_default_display_settings():
+            assets = [i for i in self.stac_data["items"] if i["id"] == items_dropdown.value][0]["assets"]
+            href = [i for i in self.stac_data["items"] if i["id"] == items_dropdown.value][0]["href"]
+                      
+            if len(assets.keys()) > 0:
+                with output:
+                    output.clear_output()
+                    # print("ASSETS", assets)
+                asset_keys = assets.keys()
+                if "data" in asset_keys:
```

Comment:
asset names aren't always consistent with "data" or "cog_default", assets can theoretically be anything descriptive like "red", "green", "blue", or "coastal" (this is why icesat2-boreal currently isn't working 😢)

Reply:
Even better, multiple assets might be COGs. Long term we need a better approach. I wonder if we can set a "cog_default" by reading the media types on the collection assets?
https://stac.maap-project.org/collections/icesat2-boreal

![image](https://github.com/MAAP-Project/stac_ipyleaflet/assets/306179/1a40c5d1-abc3-4e0c-9fc0-91b9ae07d3e1)

role = data + type = "image/tiff; application=geotiff; profile=cloud-optimized" (there are a few variants)

For now easiest thing would be to rename the "tif" asset to "data" or "cog_default"?

Reply:
> asset names aren't always consistent with "data" or "cog_default", assets can theoretically be anything descriptive like "red", "green", "blue", or "coastal" (this is why icesat2-boreal currently isn't working 😢)

I implemented the if "data" else "cog_default" logic when pivoting between the original AFRISAR and new JPL datasets - which both seemed to have different asset structures. Now that I'm seeing the latest boreal ice, I think I better understand that an item can have multiple assets associated. I've refactored to loop through each asset and within each, determine its media_type, looking for `geotiff` & `cloud-optimized` values to determine if it is compatible. Regardless, the boreal ice is still not displaying... 😞